src/contributors/02-local-development.md

```diff
@@ -1,4 +1,4 @@
-## Local Development
+# Local Development
 
 ### Install build requirements
```

Comment:
`##` 

Might as well make sure we aren't skipping any headings also.

Reply:
Imo it looks ugly with so many large headings in such a short text.